### ISSUE TITLE
pretrained problem

### ISSUE THREAD

**Issue by Weixueying** [ultralytics/yolov5#4229]
## ❔Question

What is the reason for the poor accuracy of replacing the main network with Ghostbottleneck and loading yolov5 pre-training weight? Do you need to retrain the modified network with coco data set instead of loading yOLOV5 weight directly?I also tried to load the ghost pre-training weight, and there was a model error, so I would like to ask you.

**Comment by glenn-jocher** [comment]
@Weixueying I don't understand your question. Any new network must obviously be trained on data first.

**Comment by Weixueying** [comment]
What I mean by that is that I have replaced the backbone network with Ghost and got poor training results when training with yolov5 weights, so I was wondering if I need to train on coco first and then train my own data set with the weights I got.

**Comment by glenn-jocher** [comment]
@Weixueying if you're asking about ghost training there already exists an issue thread for this in https://github.com/ultralytics/yolov5/issues/3234

**Comment by Weixueying** [comment]
ok,thank you very much.
